fv/bpf_test.go

```diff
@@ -1005,6 +1005,291 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					})
 				})
 
+				Context("Test Service type transitions", func() {
```

Comment:
I wonder what is the delta in runtime now, would it be worth to limit it to say TCP only?